core/src/mindustry/ui/Links.java

```diff
@@ -60,7 +60,7 @@ public LinkEntry(String name, String link, Drawable icon, Color color){
     private static String report(){
         String base = "https://github.com/Anuken/Mindustry/issues/new?assignees=&labels=bug";
         try{
-            return base + "&body=" + URLEncoder.encode(Strings.format("**Platform**: `@`\n\n**Build**: `@`\n\n**Issue**: *Explain your issue in detail.*\n\n**Steps to reproduce**: *How you happened across the issue, and what exactly you did to make the bug happen.*\n\n**Link(s) to mod(s) used**: `@`\n\n**Save file**: *The (zipped) save file you were playing on when the bug happened. THIS IS REQUIRED FOR ANY ISSUE HAPPENING IN-GAME, REGARDLESS OF WHETHER YOU THINK IT HAPPENS EVERYWHERE. DO NOT DELETE OR OMIT THIS LINE UNLESS YOU ARE SURE THAT THE ISSUE DOES NOT HAPPEN IN-GAME.*\n\n**Crash report**: *The contents of relevant crash report files. REQUIRED if you are reporting a crash.*\n\n---\n\n*Place an X (no spaces) between the brackets to confirm that you have read the line below.*\n- [ ] **I have searched the closed and open issues to make sure that this problem has not already been reported.**", System.getProperty("os.name"), Version.combined(), Vars.mods.list().select(LoadedMod::enabled).map(l -> l.meta.author + "/" + l.name + ":" + l.meta.version)), StandardCharsets.UTF_8.toString());
+            return base + "&body=" + URLEncoder.encode(Strings.format("**Platform**: `@`\n\n**Build**: `@`\n\n**Issue**: *Explain your issue in detail.*\n\n**Steps to reproduce**: *How you happened across the issue, and what exactly you did to make the bug happen.*\n\n**Link(s) to mod(s) used**: `@`\n\n**Save file**: *The (zipped) save file you were playing on when the bug happened. THIS IS REQUIRED FOR ANY ISSUE HAPPENING IN-GAME, REGARDLESS OF WHETHER YOU THINK IT HAPPENS EVERYWHERE. DO NOT DELETE OR OMIT THIS LINE UNLESS YOU ARE SURE THAT THE ISSUE DOES NOT HAPPEN IN-GAME.*\n\n**Crash report**: *The contents of relevant crash report files. REQUIRED if you are reporting a crash.*\n\n---\n\n*Place an X (no spaces) between the brackets to confirm that you have read the line below.*  \n- [ ] **I have searched the closed and open issues to make sure that this problem has not already been reported.**", System.getProperty("os.name"), Version.combined(), Vars.mods.list().select(LoadedMod::enabled).map(l -> l.meta.author + "/" + l.name + ":" + l.meta.version)), StandardCharsets.UTF_8.toString());
```

Comment:
Use `Strings.encode`, which is compatible with Android and doesn't throw an exception.

Reply:
changed, now its just one line, i might as well just add it inline as a link, but keeping it in a function would probably be better if in future it maybe reads the issue template file and overrides some variables, do let me know :) 🍿